spread/build/ubuntu-sbuild/task.yaml

```diff
@@ -0,0 +1,34 @@
+environment:
+    DEB_BUILD_OPTIONS: "parallel=$( nproc ) noopt ${NOCHECK}"
```

Comment:
`parallel` is overridden by `-j` below, no?

Reply:
Indeed it is.